src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/DownloadPreplannedMap/readme.md

```diff
@@ -0,0 +1,52 @@
+# Download a preplanned map area
+
+Take a map offline using an available preplanned map area.
+
+![](DownloadPreplannedMap.jpg)
+
+## Use case
+
+When a map is taken offline, a package containing basemap tiles, feature data, and other resources is created. In the preplanned workflow, the author of the web map has set up the offline packages ahead of time, which enables faster, more resource-efficient downloads compared to an on-demand workflow. Because the resources for the area are packaged once and can be downloaded many times by different users, this approach is more scalable for large organizations. To see the difference for yourself, compare this sample to the Generate an offline map sample.
+
+## How to use the sample
+
+Select a map area to take offline, then use the button to take it offline. Click 'Delete offline areas' to remove any downloaded map areas.
+
+## How it works
```

Comment:
Need a small explanation that if the preplanned map (via a MobileMapPackage) is already downloaded, it will use the 'on-device' version when you click the icon rather than re-downloading it from the web service unless you click the 'Delete offline areas' button.